src/types.rs

```diff
@@ -191,17 +188,8 @@ mod tests {
         assert_eq!(token.as_str(), "abc123");
 
         // Testing time is always racy, give it 1s leeway.
-        let expires_at = token.expires_at().unwrap();
+        let expires_at = token.expires_at();
         assert!(expires_at < expires + Duration::seconds(1));
         assert!(expires_at > expires - Duration::seconds(1));
     }
-
-    #[test]
-    fn test_deserialise_no_time() {
```

Comment:
I think it would be good to keep this test case and verify that deserialization results in an error.